src/libraries/System.Private.CoreLib/src/System/Globalization/DateTimeParse.cs

```diff
@@ -5461,7 +5461,7 @@ internal bool MatchSpecifiedWords(string target, bool checkWordBoundary, ref int
                 int targetPosition = 0;                 // Where we are in the target string
                 int thisPosition = Index;         // Where we are in this string
                 int wsIndex = target.IndexOfAny(WhiteSpaceChecks, targetPosition);
-                if (wsIndex == -1)
+                if (wsIndex < 0)
```

Comment:
not related to your changes: the array used above (`WhiteSpaceChecks`) stores two characters, is used only in one place and is stored in a `static` field. Should we replace similar usages of `string.IndexOfAny` with `span.IndexOfAny(value1, value2)`. If not, why?

Reply:
We could.  You'll need to fix up the offsets, since the span would need to be sliced, or change the code to be based off of a span stored in a local that's sliced as it's worked through.